docs/manual/creating-a-checker.tex

```diff
@@ -174,6 +186,51 @@ \section{The parts of a checker\label{creating-parts-of-a-checker}}
 \end{description}
 
 
+
+
+\section{Compiling and using a custom checker}
```

Comment:
Please add a `\label` LaTeX command before the `}`, so that the `check-labels` Ant task will pass and the Travis build will pass.  We cannot merge until the tests pass.